pkg/config/env.go

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package config
+
+import (
+	"fmt"
+	"os"
+)
+
+type ControllerEnv struct{}
+
+const (
+	webhooksSecretNameEnvVar      = "WEBHOOK_SECRET_NAME"
```

Comment:
```suggestion
	webhookSecretNameEnvVar      = "WEBHOOK_SECRET_NAME"
```
just one more nitpick. Here and below

Reply:
Snuck this one in a little earlier: 347a49464817bcddca42fe3fe1b94181302a4677